src/explore-education-statistics-admin/src/pages/prototypes/components/PrototypeEditableContent.tsx

```diff
@@ -95,16 +102,21 @@ export default class PrototypeEditableContent extends React.Component<
           </div>
         ) : (
           // eslint-disable-next-line jsx-a11y/no-static-element-interactions,jsx-a11y/click-events-have-key-events
-          <div className="editable-content" onClick={this.setEditing}>
-            {unsaved ? (
-              <div className="editable-button unsaved">Click to edit</div>
-            ) : (
-              <div className="editable-button">Click to edit</div>
-            )}
-            <div ref={this.ref} />
+          <div
+            className={`editable-content ${unsaved ? 'unsaved' : ''}`}
+            onClick={this.setEditing}
+          >
+            <div className="editable-button" />
+            <div
+              ref={ref => {
+                this.ref = ref;
```

Comment:
We should probably be preferring to use `createRef`s rather than callback refs. The React documentation on refs generally promotes using `createRef` before callbacks and is more consistent with the majority of the codebase.

There is also this [caveat](https://reactjs.org/docs/refs-and-the-dom.html#caveats-with-callback-refs) to their usage.